### ISSUE TITLE
Missing talkback explanation for Timetable/bookmark icon button

### ISSUE THREAD

**Issue by fumiya-kume** [DroidKaigi/conference-app-2023#462]
**Describe the bug**
Missing talkback explanation for Timetable/bookmark icon button

The contentDescription should have.

**To Reproduce**
1. Go to Timetable screen
2. Tap bookmark button

**Expected behavior**

When the attendees tap the bookmark button, should explains about button feature / type etc...

**Screenshots**

<img src="https://github.com/DroidKaigi/conference-app-2023/assets/16269075/e8383c24-7ba5-4596-8f7a-4fb968d33a8b" width="50%" />

**Comment by takahirom** [comment]
We should use contentDescription attribute 👍

**Comment by Pluu** [comment]
🙋

**Comment by takahirom** [comment]
Thanks! Assigned 👍
